### PR TITLE
Fix Issue #62 - Data Persistence Between Sessions

### LINKED ISSUE #62: Data Doesn't Persist Between Sessions
Need to handle session startup and loading data into memory rather than removing it.

### PR DESCRIPTION
**Pull request by FrancescoSTL**
Fixes #62

### CODE REVIEW

**Comment by FrancescoSTL** [comment]
@purukaushik r? When you get the chance

File: web-crawler.js [hunk 1]
```diff
@@ -234,6 +203,50 @@ function startRequestListeners() {
...
+            sendResponse({ "deletedOverview": true})
+        } else if (!profiling) { // if we are supposed to stop profiling
+            var JSONString = stringifyAssetStore(profileStorage, false);
+            // send the profiling data
+            sendResponse({ "profiles": JSONString });
+
+            // and clear the profiling storage
+            profileStorage.clear();
+        }
+    });
```
Comment (purukaushik) [inline web-crawler.js:263]: each `request.condition` must be in separate if-loops and each exiting from a `return;`

**Comment by purukaushik** [comment]
just one comment ^^ . otherwise good to merge

**Comment by FrancescoSTL** [comment]
donezo

**Pull request merged**